emeis/core/views.py

```diff
@@ -12,6 +14,14 @@ def get_queryset(self):
         queryset = super().get_queryset()
         return queryset.model.visibility_queryset_filter(queryset, self.request)
 
+    def destroy(self, request, *args, **kwargs):
```

Comment:
What is destroy? Looked into django, but it took too long.

Reply:
It's called on `DELETE` requests.